subprojects/platform-jvm/src/integTest/groovy/org/gradle/jvm/toolchain/SharedJavaInstallationRegistryIntegrationTest.groovy

```diff
@@ -66,7 +66,7 @@ class SharedJavaInstallationRegistryIntegrationTest extends AbstractIntegrationS
 
                 void apply(Project project) {
                     project.tasks.register("show") {
-                        println registry.listInstallations()
+                       registry.listInstallations().each { println it }
```

Comment:
Just realising that this will cause the printing at configuration time and not execution time.

Reply:
Yes, is that a problem for the test?

Reply:
No :-)